modules/reference/pages/properties/topic-properties.adoc

```diff
@@ -525,6 +525,23 @@ NOTE: Although `replication.factor` isn't returned or displayed by xref:referenc
 
 ---
 
+[[deleteretentionms]]
+==== delete.retention.ms
+
+The retention time for tombstone records in a compacted topic. Redpanda removes tombstone records after the retention limit is exceeded.
+
+If you have enabled Tiered Storage and set <<redpandaremoteread,`redpanda.remote.read`>> or <<redpandaremotewrite,`redpanda.remote.write`>> for the topic, you cannot enable tombstone removal. 
+
+If both `delete.retention.ms` and the cluster property `tombstone_retention_ms` are set, `delete.retention.ms` overrides the cluster level tombstone retention for an individual topic.
+
+**Default**: null
```

Comment:
@WillemKauf I'm asking @Deflaimun to pull the cluster config reference description so we can add it to the docs, but afaict there is no default value for `tombstone_retention_ms` either. Is there a value that we recommend users set? Or perhaps we can provide some guidance or scenarios users can look out for so they can set a tombstone retention policy that works well for them?

Reply:
That's right, both are set to `null` by default in order to reduce configuration headaches with tiered storage property conflicts.

Confluent/Kafka default this value to one day (24 hours, `86400000` ms).

Guidance/scenarios:

* If more frequent cleanup of tombstone records is important for space management, this value may be reduced.
* If obtaining a complete snapshot of the log, including tombstone records, is important to your consumers, this value should be set such that consumers have enough time for their reads to complete before tombstones are removed.